JavaAssignment/src/arpit/AdditionQ6.java

```diff
@@ -0,0 +1,54 @@
+package arpit;
+
+public class AdditionQ6 {
+
+	int sum1;
+	double sum2;
+	String sum3;
+
+	// This function is perform addition of two integer number and return
+	// result.
+	int addingInteger(int num1, int num2) {
+
+		sum1 = num1 + num2;
+		return sum1;
+	}
+
+	// This function is perform addition of two double number and return result.
+	double addingDouble(double num1, double num2) {
+
+		sum2 = num1 + num2;
+		return sum2;
+
+	}
+
+	// This function is perform addition of two String value and return result.
+	String addingString(String S1, String S2) {
+
```

Comment:
Arpit, here you need to parse String value & after that you need to return that value in String format

Reply:
Thank you Maulik for your comment. I will update the code and revert back to you.